source/api.rst

```diff
@@ -185,10 +185,10 @@ Possible scenarios when this channel notifies are: the transaction is announced
 
 **unconfirmedRemoved/{address}**
 
-The unconfirmedRemoved channel notifies when a transaction related to an address had the unconfirmed state, but not anymore.
-The messages returned contain the transactions hashes.
+The ``unconfirmedRemoved`` channel notifies subscribed clients when a transaction related to the given address exits the unconfirmed state.
```

Comment:
perhaps `a transaction related to the given address is removed from the unconfirmed state`?

Reply:
I think of "states" as something you enter and leave, not as a group or pool from which you can be added or removed. And certainly not as labels you can have, or not have.
Therefore I think enters/leaves the unconfirmed state is easier to understand.
But I do not feel very strongly about this so feel free to insist if you do :)